doc/source/_static/version_switcher.json

```diff
@@ -1,9 +1,14 @@
 [
     {
-        "name": "1.9.0 (dev)",
+        "name": "dev",
         "version": "dev",
         "url": "https://scipy.github.io/devdocs/"
     },
+    {
+        "name": "1.9.0 (stable)",
+        "version":"1.9.0",
+        "url": "https://docs.scipy.org/doc/scipy-1.9.0/"
+    },
     {
         "name": "1.8.1 (stable)",
```

Comment:
looks like you should remove `(stable)` here?

Reply:
ooops yes